docs/source/developer_documentation.rst

```diff
@@ -0,0 +1,131 @@
+Developer Documentation
+=======================
+
+**Note this document is a work in progress. Contrbitions to all sections are welcome.**
+
+Github Workflow
+---------------
+
+-  You should request reviews for pull requests from ``multisignal-epi-inference-devs``, unless you have a good reason to request reviews from a smaller group.
+-  Reviews from all of ``multisignal-epi-inference-devs`` are  encouraged, but an approving review from a `codeowner <https://github.com/CDCgov/multisignal-epi-inference/blob/main/.github/CODEOWNERS>`_ (@dylanhmorris, @damonbayer, or @gvegayon) is required before a pull request can be merged to ``main``.
+-  For CDC contributors, If your pull request has not received a review at the time of the next standup, use standup to find a reviewer.
+-  External contributors should expect to receive a review within a few days of creating a pull request.
+-  If you create a draft pull request, indicate what, if anything, about the current pull request should be reviewed.
+-  Only mark a pull request as “ready for review” if you think it is ready to be merged. This indicates that a thorough, all-encompassing review should be given.
+
+Installation for Developers
+---------------------------
+
+-  poetry install with dev dependencies
+-  pre-commit install
+
+Coding Conventions
+------------------
+
+A variety of coding conventions are enforced by automated tools in continuous integeration ([black](https://github.com/psf/black), [isort](https://github.com/PyCQA/isort), [ruff](https://github.com/astral-sh/ruff), [numpydoc-validation](https://github.com/numpy/numpydoc)) via [pre-commit](https://github.com/pre-commit/pre-commit) hooks.
```

Comment:
These are markdown links; need conversion to `.rst`

Reply:
addressed in https://github.com/CDCgov/multisignal-epi-inference/pull/224/commits/03ddb0a85769f5e2c7c6c6b7aa2f99c3893800df